src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Totally agree with everything you say. Thank you for making the exception (for my mistake) in this case though.